README.md

```diff
@@ -210,6 +210,35 @@ console.log(
 );
 ```
 
+## File Size Parsing
+The `FileSizeFormat` object can be used to format a file size appropriately for the user's locale.
+
+Syntax:
+
+```javascript
+var formatFS = new d2lIntl.FileSizeFormat(locale[, options]);
+```
+Options:
+
+- **locale**: see [overriding locale data](#Overriding Locale Data)
+- **maximumFractionDigits**: Change the default maximum amount of decimal places to include in the result (Default: 2)
```

Comment:
As noted below, I don't think this option is necessary, let's just use a good default and add it later if someone really needs it.